android/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These keep getting generated for some reason and no .gitignore for them?
I will delete the files